patches/tModLoader/Terraria/ModLoader/ChestLoot.Condition.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using Terraria.Enums;
+using Terraria.GameContent.Events;
+using Terraria.Localization;
+
+namespace Terraria.ModLoader;
+
+public partial class ChestLoot {
+	public interface ICondition {
+		string Description { get; }
+
+		bool IsAvailable();
+	}
+
+	public sealed class Condition : ICondition {
+		#region Common conditions
+		public static readonly Condition Christmas = new(NetworkText.FromLiteral("During Christmas"), () => Main.xMas);
```

Comment:
Need to add localizations for all of these. No `FromLiteral`

Intent is to change `Description` from `string` and `NetworkText` to `LocalizedText` soon